spec/features/profile_management_spec.rb

```diff
@@ -47,13 +86,36 @@
     expect(page).to have_content "Name: can't be blank"
   end
 
-  specify "can delete a profile" do
-    visit profiles_path
+  context "destroy" do
+    specify "can delete a profile" do
+      visit profiles_path
 
-    click_link "Delete"
+      within "#profile_#{profile.id}" do
+        click_link "Delete"
+      end
+
+      expect(page).to have_content "Profile successfully deleted"
+      expect(page).to_not have_content profile.name
+    end
```

Comment:
Have decided to not test this as it's much harder using Poltergeist than either Selenium or Capybara-Webkit. Manually tested in a browser.